### DISCUSSION TITLE
GPG Key

### DISCUSSION THREAD

**Discussion by tacerus** [fish-shell/fish-shell#8894] [General]
Hi,

is one of these the correct GPG Key to verify the `*.asc` signatures against? I could not find one on the fish website, hence searched a keyserver for the ID given in the release notes.

https://keyserver.ubuntu.com/pks/lookup?search=0x7A67D962D88A709A&fingerprint=on&op=index

Thanks!

**Comment by faho** [comment]
The gpg key that signs the releases is the same that signs the announcement mails: https://sourceforge.net/p/fish/mailman/message/37630685/

**Reply by tacerus** [reply]
Thank you very much!
